src/cogs/jaegeraccounts.py

```diff
@@ -209,14 +211,17 @@ class AccountDistrubution(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+    def _account_to_sheet(self, url, account):
+        gspread_service_account.open_by_url(url).sheet1
+
     @commands.guild_only()
     @commands.group(invoke_without_command=True)
     async def account(self, ctx):
-        """Prints the currently booked account."""
         async with dbPool.acquire() as conn:
             url = await conn.fetchval("SELECT url FROM sheet_urls WHERE fk = (SELECT id FROM guilds WHERE guild_id = $1);", ctx.guild.id)
         if url is None:
-            raise NoSheetsUrlException("There is no google sheets url associated with this guild.")
+            raise NoSheetsUrlException(
+                "There is no google sheets url associated with this guild.")
```

Comment:
Looks like autoformatting, either on my old code or on your end.
```suggestion
            raise NoSheetsUrlException("There is no google sheets url associated with this guild.")
```